_posts/2020/2020-05-15-constraints-and-freedom-writing-perfect-program.md

```diff
@@ -113,7 +112,7 @@ Please enter the second real number
 Sum of the numbers = 3.3000000000000003
 ```
 
-The precision is actually working as intended, as the Java 8 [tutorial](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html){:target="_blank"} notes that *this data type should never be used for precise values*.
+The precision was actually working as intended, as the Java 8 [tutorial](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html){:target="_blank"} notes that *this data type should never be used for precise values*.
```

Comment:
Hm. This is more like a side note, as this is the intended behavior of something. Yes, it _was_ working before, but it's supposed to _still_ work now